src/sage/arith/misc.py

```diff
@@ -3498,11 +3497,15 @@ def crt(a, b, m=None, n=None):
 CRT = crt
 
 
-def CRT_list(values, moduli):
+def CRT_list(values, moduli=None):
     r""" Given a list ``values`` of elements and a list of corresponding
     ``moduli``, find a single element that reduces to each element of
     ``values`` modulo the corresponding moduli.
```

Comment:
```suggestion
def CRT_list(values, moduli=None):
    r"""
    Given a list ``values`` of elements and a list of corresponding
    ``moduli``, find a single element that reduces to each element of
    ``values`` modulo the corresponding moduli.
```
This is a while-we-are-at-it thing to fix.